stream-chat-android-ui-components/src/main/res/layout/stream_ui_channel_list_item_view.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/channelItemView"
```

Comment:
Temp solution for this sample test for focus on the setup and not on writing custom matchers.